### DISCUSSION TITLE
user_setup.h how to use

### DISCUSSION THREAD

**Discussion by suntchi2018** [Bodmer/TFT_eSPI#3181] [General]
Hi Bodmer,
one question, I apologize in advance for my lack of experience, but since each project has its own particular setup for the display, it is possible to customize the setup by also inserting the settings file in the program folder without having to modify the user_setup .h file every time in the library folder?
In this way each project has its own display configuration file present in the program folder.
At the moment I modify the user_setup.h of the library every time, I tried to insert it in the program folder, but doing so doesn't work.

**Comment by PaulVdBo** [comment]
Same question here...

**Reply by suntchi2018** [reply]
Many thanks, sorry me

**Comment by suntchi2018** [comment]
I read the instructions, sorry for the inconvenience...
Just want to say Bodmer that your library is fantastic and your support is superb, thank you
